# SEDDOM config for clapper generated dataset

# Map configuration
resolution: 0.3
semantic_class: semantic_kitti # Type of semantic class
random_samples_per_beam: -1
free_resolution: -1 # Free space sampling resolution
ds_resolution: 0.1 # Downsampling factor
chunk_depth: 4
max_range: -1 # Sensor max range (m)
block_depth: 3 # This is defined in compile time

# Kernel parameters
sf2: 10.0 # Actually sigma_0 in sparse kernel
ell: 0.3 # Length scale of the sparse kernel

# BGK Inference class prior
prior: 0.001 # each class
